vips/title_of_new_vip.md

```diff
@@ -0,0 +1,132 @@
+---
+VIP: 
+Title: The Ultimate State Pruner
+Description: This proposal is based on VIP-211 and aims to achieve efficient state pruning by utilizing new nodes to mask old nodes.
+Author: Some Author(author@foo.com)
+Discussions: <URL>
```

Comment:
preamble header `discussions` must be a URL.

Reply:
Updated